catalog/core/catalog-core-camelcomponent/src/main/java/ddf/camel/component/catalog/content/ContentProducer.java

```diff
@@ -187,4 +161,20 @@ public void process(Exchange exchange)
         LOGGER.debug("EXITING: process");
     }
 
+    private Map<String, String> createAttributeOverrideMapFromHeaders(
+            String contentDirectoryHeaders) {
+        Map<String, String> attributeOverrideMap = null;
+        if (StringUtils.isNotBlank(contentDirectoryHeaders)) {
+            attributeOverrideMap = new HashMap<>();
+            String[] attribute = contentDirectoryHeaders.split(",");
+            for (int i = 0; i < attribute.length; i++) {
+                String[] keyValuePair = attribute[i].split("=");
+                if (keyValuePair.length == 2) {
+                    attributeOverrideMap.put(keyValuePair[0], keyValuePair[1]);
+                }
+            }
+        }
+        return attributeOverrideMap;
```

Comment:
This should be a non issue.  A null attributeOverrideMap is handled in the create call in the catalogFramework.